data/ooniprobe.1

```diff
@@ -119,6 +121,9 @@ Useful when debugging freezes or locks in complex code.
 .TP
 .BR \-\-version
 Display the ooniprobe version and exit.
+.TP
+.BR \-\^Q " or " \-\-queue
+AMQP Queue URL amqp://user:pass@host:port/vhost/queue
```

Comment:
In sort this is used to make ooniprobe work with blocked: https://github.com/openrightsgroup/cmp-issues/issues/78
Not sure how we can describe this in a way that makes senses to the user.